src/utils/sdk-utils.ts

```diff
@@ -23,18 +23,20 @@ export async function isAgentRunning() {
   }
 
 export async function postSnapshot(body: any) {
-    const url = `http://localhost:${Constants.PORT}${Constants.SNAPSHOT_PATH}`
+    const URL = `http://localhost:${Constants.PORT}${Constants.SNAPSHOT_PATH}`
+    const ONE_HUNDRED_MB_IN_BYTES = 100_000_000
+
     return Axios({
       method: 'post',
-      url,
+      maxContentLength: ONE_HUNDRED_MB_IN_BYTES,
+      url: URL,
       data: body,
     } as any).then(() => {
       return true
-    }).catch((error) => {
+    }).catch(() => {
       // This code runs in the context of the SDK, so the SDK needs to have LOG_LEVEL=debug
       // enabled for these logs to appear.
-      logger.debug(`Error posting snapshot to ${url} with body: ${body}`)
-      logger.debug(error)
+      logger.debug(`Error posting snapshot to ${URL}`)
```

Comment:
Would it be worth having this as a log that isn't hidden behind `LOG_LEVEL=debug`?

Reply:
Yup. Will update.

Reply:
I changed it to use `logError`, which is a central way we have been rendering errors. It's defined here https://github.com/percy/percy-agent/blob/d01479d4387e5a9350199d163e0f387a835dab87/src/utils/logger.ts#L29-L32

It explicitly logs an error and then puts the full dump in debug. It outputs in the log like this (without `LOG_LEVEL=debug` being set)

```
    with large dom
[percy] Error Request body larger than maxBodyLength limit
[percy] Error posting snapshot to agent.
      ✓ snapshots without error (2002ms)
```